.github/workflows/on-push-to-release.yml

```diff
@@ -0,0 +1,120 @@
+name: On push to main
+
+on:
+  push:
+    branches: [ main ]
+
```

Comment:
This file is called `on-push-to-release.yml`, so these are typos right? We don't need this workflow when using release please as that's based on the main branch.

Reply:
That is right